docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -951,6 +952,19 @@ The `--disable-multiple-windows` option disables support for multi-window testin
 
 *Related configuration file property*: [disableMultipleWindows](configuration-file.md#disablemultiplewindows).
 
+### --retry-test-pages
+
+If this option is enabled, TestCafe retries failed network requests for webpages it navigates. The retry limit is ten times.
```

Comment:
Can TestCafe "navigate the webpages"? Browsers can probably "navigate the Internet" (as in "navigate the seas") but can something "navigate the webpages"? Or does it "navigate **to** the webpages"?

For the `--client-scripts` option, we say "pages visited during the tests". Maybe we should use the same expression for both options? (Not necessarily "pages visited during the tests" if you can come up with a better one).

Reply:
`If this option is enabled, TestCafe retries failed network requests for webpages visited during tests.`

Do we need `internally` in the sentence below? Which one (do you think) is better:

`This feature uses Service Workers internally and requires a secure connection.`
vs
`This feature uses Service Workers  and requires a secure connection.`

Reply:
Do we need `internally` in the sentence below? Which one (do you think) is better:

`This feature uses Service Workers internally and requires a secure connection.`
vs
`This feature uses Service Workers  and requires a secure connection.`